src/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -122,6 +122,17 @@ class Telemetry : public PluginBase {
         float yaw_rad_s; /**< @brief Yaw angular velocity */
     };
 
+    /**
+     * @brief Ground speed type.
+     *
+     * Metrics typically displayed on a HUD for fixed wing aircraft.
+     */
+    struct VfrHud {
+        float airspeed_m_s; /**< @brief Current indicated airspeed (IAS) in metres/second. */
+        float throttle; /**< @brief Current throttle setting (0 to 100). */
```

Comment:
Is it really from 0 to 100? And not 0 to 1? In that case it should be called `throttle_percentage`.

Reply:
Ok, no problem on that, i'm changing it